packages/frontend-2/components/common/Card.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="border border-outline-3 rounded-lg p-5 flex flex-col">
+  <div
+    class="border border-outline-3 rounded-lg flex flex-col"
+    :class="size === 'sm' ? 'p-3' : 'p-5'"
```

Comment:
@andrewwallacespeckle Do we even use p-5 anywhere? I feel like basically we use p-3 everywhere?

Reply:
Cool, let's see if we can align them as some point :)